templates/taxi/car_detail.html

```diff
@@ -19,4 +19,13 @@ <h1>Drivers</h1>
       <li>{{ driver.username }} ({{ driver.first_name }} {{ driver.last_name }})</li>
     {% endfor %}
   </ul>
+    {% if user in car.drivers.all %}
```

Comment:
Similarly, `car.drivers.all` should be called with parentheses as `car.drivers.all()` to ensure proper execution of the method.